Index: .idea/inspectionProfiles/Project_Default.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/inspectionProfiles/Project_Default.xml b/.idea/inspectionProfiles/Project_Default.xml
new file mode 100644
--- /dev/null	(date 1706029045703)
+++ b/.idea/inspectionProfiles/Project_Default.xml	(date 1706029045703)
@@ -0,0 +1,31 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="NonAsciiCharacters" enabled="false" level="WARNING" enabled_by_default="false" />
+    <inspection_tool class="PyPep8Inspection" enabled="true" level="WEAK WARNING" enabled_by_default="true">
+      <option name="ignoredErrors">
+        <list>
+          <option value="E741" />
+        </list>
+      </option>
+    </inspection_tool>
+    <inspection_tool class="PyPep8NamingInspection" enabled="true" level="WEAK WARNING" enabled_by_default="true">
+      <option name="ignoredErrors">
+        <list>
+          <option value="N802" />
+          <option value="N803" />
+        </list>
+      </option>
+    </inspection_tool>
+    <inspection_tool class="PyUnresolvedReferencesInspection" enabled="true" level="WARNING" enabled_by_default="true">
+      <option name="ignoredIdentifiers">
+        <list>
+          <option value="float.split" />
+          <option value="list.strip" />
+          <option value="int.strip" />
+          <option value="dict.*" />
+        </list>
+      </option>
+    </inspection_tool>
+  </profile>
+</component>
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1706029045744)
+++ b/.idea/vcs.xml	(date 1706029045744)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1706029074869)
+++ b/.idea/misc.xml	(date 1706029074869)
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="Black">
+    <option name="sdkName" value="Python 3.11" />
+  </component>
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.11" project-jdk-type="Python SDK" />
+</project>
\ No newline at end of file
Index: .idea/inspectionProfiles/profiles_settings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/inspectionProfiles/profiles_settings.xml b/.idea/inspectionProfiles/profiles_settings.xml
new file mode 100644
--- /dev/null	(date 1706029045762)
+++ b/.idea/inspectionProfiles/profiles_settings.xml	(date 1706029045762)
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="USE_PROJECT_PROFILE" value="false" />
+    <version value="1.0" />
+  </settings>
+</component>
\ No newline at end of file
Index: Python - Módulos Mundo 03/ex.110/utilidadescev/dados/__init__.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\ndef leiaDinheiro(msg):\r\n    while True:\r\n        valor = input(msg).replace(',', '.')\r\n        if valor.replace('.', '', 1).isdigit():\r\n            return float(valor)\r\n        else:\r\n            print('\\033[1;31mValor inválido. Digite um valor monetário válido.\\033[m')\r\n\r\n\r\n\r\n\r\ndef leiaInt(número):\r\n    while True:\r\n        try:\r\n            print(número, end='')\r\n            número = int(input())\r\n            return número\r\n\r\n        except ValueError:\r\n            print('\\033[1;31mERRO! Digite um número inteiro válido.\\033[m')\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Python - Módulos Mundo 03/ex.110/utilidadescev/dados/__init__.py b/Python - Módulos Mundo 03/ex.110/utilidadescev/dados/__init__.py
--- a/Python - Módulos Mundo 03/ex.110/utilidadescev/dados/__init__.py	(revision faac4b70fbdce11ec4a353ec22fbed472b06ba4b)
+++ b/Python - Módulos Mundo 03/ex.110/utilidadescev/dados/__init__.py	(date 1706029074273)
@@ -8,8 +8,6 @@
             print('\033[1;31mValor inválido. Digite um valor monetário válido.\033[m')
 
 
-
-
 def leiaInt(número):
     while True:
         try:
Index: Python - Módulos Mundo 03/ex.112.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Python - Módulos Mundo 03/ex.112.py b/Python - Módulos Mundo 03/ex.112.py
new file mode 100644
--- /dev/null	(date 1706029126727)
+++ b/Python - Módulos Mundo 03/ex.112.py	(date 1706029126727)
@@ -0,0 +1,1 @@
+# Exercício 114: Crie um código em Python que teste seo site Pudim está acessível pelo computador usado.
diff --git a/Python - Módulos Mundo 03/ex.110/ex.111.py b/Python - Módulos Mundo 03/ex.111.py
rename from Python - Módulos Mundo 03/ex.110/ex.111.py
rename to Python - Módulos Mundo 03/ex.111.py
